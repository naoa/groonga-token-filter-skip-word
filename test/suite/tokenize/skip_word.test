register token_filters/skip_word

table_create skip_words TABLE_HASH_KEY ShortText
load --table skip_words
[
{"_key": "and"},
{"_key": "or"}
]

tokenize TokenBigram "Hello and World" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterSkipWord
