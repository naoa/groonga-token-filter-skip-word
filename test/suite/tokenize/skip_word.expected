register token_filters/skip_word
[[0,0.0,0.0],true]
table_create skip_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table skip_words
[
{"_key": "and"},
{"_key": "or"}
]
[[0,0.0,0.0],2]
tokenize TokenBigram "Hello and World"   --normalizer NormalizerAuto   --token_filters TokenFilterSkipWord
[[0,0.0,0.0],[{"value":"hello","position":0},{"value":"world","position":2}]]
